front/app/templates/application.hbs

```diff
@@ -31,3 +28,5 @@
     {{outlet}}
 
 </div>
+<script> var $buoop = {vs:{i:11,f:-4,o:-4,s:9,c:-4},unsecure:true,mobile:false,api:4}; function $buo_f(){  var e = document.createElement("script");  e.src = "//browser-update.org/update.min.js";  document.body.appendChild(e);};try {document.addEventListener("DOMContentLoaded", $buo_f,false)}catch(e){window.attachEvent("onload", $buo_f)}</script>
+
```

Comment:
@dikaiosune You are right, it is from an outdated diff, my bad! This seems like a good approach though, small enough to be unobtrusive, but still there to let the user know that things might be broken if they continue to use their browser.

I think the ember-cli plugin may be the preferable implementation of this: https://www.npmjs.com/package/ember-cli-browser-update but does require more work and the result would be the same either way. This PR already integrates the tag, so I don't see any reason not to use it, and the settings look good.